src/js/containers/AppContainer.jsx

```diff
@@ -21,7 +21,7 @@ import PendingPage from '../components/login/PendingPage';
 let devExtension;
 if (kGlobalConstants.DEV || kGlobalConstants.LOCAL) {
     // only enable Redux debugging in dev and local modes
-    devExtension = window.devToolsExtension ? window.devToolsExtension() : undefined;
+    devExtension = window.__REDUX_DEVTOOLS_EXTENSION__ ? window.__REDUX_DEVTOOLS_EXTENSION__() : undefined;
 }
 
```

Comment:
@ebdabbs ,

This is likely beyond the scope of this PR, but would like to dialogue at some point about the pros/cons of including the extension in Production.

Just read over a bit of [this article](https://medium.com/@zalmoxis/using-redux-devtools-in-production-4c5b56c5600f) which I think is generally in favor of doing so, with little to zero extra configuration required!